CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
+## 1.17.2
+### July 10, 2024
+
+CHANGES:
+
+* core: Bump Go version to 1.22.5
+* secrets/azure: Update plugin to v0.19.2 [[GH-27652](https://github.com/hashicorp/vault/pull/27652)]
+
+FEATURES:
+
+* **AWS secrets engine STS session tags support**: Adds support for setting STS
```

Comment:
We don't typically add "New Features" in minor releases so this one looks a little weird to me.

Reply:
I see this went into 1.17.2 https://github.com/hashicorp/vault/pull/27620

Reply:
I think we should not have backported the feature